### ISSUE TITLE
Add search feature to Private Cloud WordPress

### ISSUE THREAD

**Issue by ty2k** [BCDevOps/developer-experience#2952]
Our [Private Cloud WordPress site](https://cloud.gov.bc.ca/private-cloud/) currently has the built-in WordPress search form hidden with CSS. Add the search form back and see if it's possible/easy to add it to the right-hand side of the main menu navigation as indicated with the red circle here:

![Desired location of the search bar circled in red](https://images.zenhubusercontent.com/5ec709daacce4878764898c4/c54256be-8440-460e-96c6-3b1bcc81df39)

**Rational/Why**
As a user, I want to be able to use a site-wide search to quickly search for pages. Right now, I don't have that ability, and have to explore the menus to find what I'm looking for.

**Additional Context** 
- Private Cloud WordPress site: https://cloud.gov.bc.ca/private-cloud/
- Code repository for code that gets added to the WordPress site through the UI: https://github.com/bcgov/platform-services-wordpress
- The code that's hiding the search form gets added within the "Additional CSS" menu in WordPress
- Currently, authenticated users can search the WordPress site using the upper WordPress UI that shows their user name and links to the dashboard, etc.

**Dependencies**
- The Private Cloud WordPress site itself is the only dependency.

**Tests**
- Perform smoke tests on the WordPress search engine with some terms like "openshift 101", "support", etc to see if the results are sane. If search results are very bad, write a new ticket for exploring better search solutions.

**Definition of done/Acceptance Criteria**
- [x] Search appears visually on the site and is usable
- [x] A non-authenticated user can visit the site, enter a search into the text input field, and perform a search by clicking a button or hitting return.
- [x] `platform-services-wordpress` code repository is updated with latest CSS styles

**Comment by ty2k** [comment]
I've updated the [Private Cloud WordPress site](https://cloud.gov.bc.ca/private-cloud/) with styling to show the search form. Unfortunately, the positioning of this component in the theme markup is not something we are able to change, and we are not able to position the search form within the main menu navigation.

Search form on homepage:
![Search form visible on homepage](https://images.zenhubusercontent.com/5ec709daacce4878764898c4/fe53313d-21a3-4e1b-8ed4-71879a5e3fc7)

Search results page:
![Search results page](https://images.zenhubusercontent.com/5ec709daacce4878764898c4/277dac33-d3e3-44d6-99c0-2096b152ad93)

The code repository where we keep our CSS for this site [has been updated](https://github.com/bcgov/platform-services-wordpress/pull/7).
